src/App.js

```diff
@@ -10,7 +10,7 @@ import ActionCable from 'actioncable'
 import './App.css';
 import {deAuthenticate} from "./Modules/Auth";
 
-const cable = ActionCable.createConsumer("ws://localhost:3001/cable");
+const cable = ActionCable.createConsumer("https://planted-api.herokuapp.com/cable");
```

Comment:
;-) What makes you think you can serve WebSockets through http?

Reply:
:( can fix :)